sigstore-gradle/sigstore-gradle-sign-base-plugin/src/main/kotlin/dev/sigstore/sign/SigstoreSignExtension.kt

```diff
@@ -39,7 +39,7 @@ abstract class SigstoreSignExtension(private val project: Project) {
     abstract val sigstoreJavaVersion : Property<String>
 
     init {
-        sigstoreJavaVersion.convention("0.5.0")
+        sigstoreJavaVersion.convention("0.4.2")
```

Comment:
Unless we release `sigstore-java` as well on Maven Central, this should remain `0.4.0`